frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -76,6 +78,21 @@ const validateUserTemplate: VmSettingsValidator = (field, options) => {
   const { getState, id } = options;
   const state = getState();
 
+  const forcedSingleUserTemplateName = iGetCommonData(state, id, VMWizardProps.userTemplateName);
```

Comment:
IMO this error can happen in every flow, so we don't have to depend on `forcedSingleUserTemplateName`